Sources/MapboxCoreNavigation/RerouteController.swift

```diff
@@ -0,0 +1,211 @@
+import Foundation
+import MapboxDirections
+@_implementationOnly import MapboxNavigationNative_Private
+
+class RerouteController {
```

Comment:
“Reroute” is a verb, so this class is named like a method. “ReroutingController” would be a more natural name for a class. On the other hand, if this class is purely intended to wrap `MapboxCoreNavigation.RerouteController`, when there should be a comment to that effect so we don’t expose this class name publicly in the future.

Reply:
Added code doc to clarify the purpose of this class